.vscode/settings.json

```diff
@@ -1,6 +1,13 @@
 {
+  "tailwindCSS.rootFontSize": 10,
   "typescript.tsdk": "node_modules/typescript/lib",
-  "tailwindcss.jsxPropImportChecking": false,
-  "tailwindcss.rootFontSize": 10,
-  "typescript.preferences.importModuleSpecifier": "non-relative"
+  "typescript.preferences.importModuleSpecifier": "non-relative",
+  "tailwindCSS.experimental.classRegex": [
+    ["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"],
+    ["cx\\(([^)]*)\\)", "(?:'|\"|`)([^']*)(?:'|\"|`)"]
+  ],
```

Comment:
Should we add a note to Readme / code / somewhere else that autocomplete might be a little flaky sometimes?